### DISCUSSION TITLE
Should we ditch lfc -r (rebuild)?

### DISCUSSION THREAD

**Discussion by cmnrd** [lf-lang/lingua-franca#528] [General]
I am trying to fix the rebuild option of lfc (see #521 and #524). While working on this, it occurred to me that the rebuild causes quite some trouble. While we can certainly fix the discussed issues, the whole thing seems to be pretty fragile. **And** we need to be aware that this mechanism only works in a developer setup. If we take a pre-built lfc and run `lfc -r` bad things will happen.

I am thinking that we should let gradle handle the building altogether (it's what it is made for after all). We already have the `buildLfc` target for gradle. So instead of using `lfc -r` we could simply do `./gradlew --runLfc --args`, which will rebuild lfc and run it.

What do you think?

**Comment by lhstrh** [comment]
As per https://github.com/lf-lang/lingua-franca/commit/84efdc885090483b71e26eb56425ef9f8fe153d0, the rebuild feature no longer exists. We added a `-r`/`--run` flag to `build-lfc` instead. This allows one to rebuild and then subsequently run `lfc` with whatever arguments come after the `-r` switch. 

For instance, `build-lfc -r --help` will build and then invoke `lfc --help`.
